src/xalanc/PlatformSupport/XalanFileOutputStream.hpp

```diff
@@ -40,8 +40,9 @@
 
 namespace XALAN_CPP_NAMESPACE {
 
-
+#if !defined(XALAN_WINDOWS)
```

Comment:
Why is this inappropriate for Windows?

Reply:
The Windows one seems to be using HANDLE (i.e. void*) type instead of FILE type. Also with mingw clang this happens error: no member named 'FILE' in namespace 'std'; did you mean '::FILE'.